### ISSUE TITLE
How to handle island territories in `shift_geometry()`?

### ISSUE THREAD

**Issue by walkerke** [walkerke/tigris#122]
The documentation of `shift_geometry()` in the newly released version of tigris reads: 

> The function is also designed to work exclusively with features in the continental United States, Alaska, Hawaii, and Puerto Rico. If your dataset includes features outside these areas (e.g. other US territories or other countries), you may get unworkable results. It is advisable to filter out those features before using `shift_geometry()`.

The documentation's examples also use the argument `resolution = "20m"` which should almost always be used for small-scale mapping of the entire United States; this dataset omits American Samoa, Guam, Northern Mariana Islands, and the US Virgin Islands.  However, users may attempt to use `shift_geometry()` without reading this and get odd results:

```r
library(tigris)

st <- states() %>%
  shift_geometry()

plot(st$geometry)
```

![image](https://user-images.githubusercontent.com/4872833/118539100-a1665800-b714-11eb-832e-0be4c0a1cf3e.png)

The long archipelago of uninhabited islands NW of Hawaii is also included, which isn't addressed when the US territories are filtered out.  I should probably think about a way to handle this to avoid confusing users; posting this issue for consideration for tigris 1.5.

**Comment by walkerke** [comment]
From @benjaminrobinson in #123: 

> Good afternoon, Dr. Walker!
Thank you so much for your work on this package and so many others. Having a function like shift_geometry makes a lot of my job so much easier when I want to make a map that includes Hawaii, Alaska, and Puerto Rico in R. I've also been relatively successful in convincing some of the analysts in my company's GIS shop to switch over to R from Python specifically because the package environment is so robust and your package(s) play a large role in that!

My organization also wants to make maps that include the U.S. territories (i.e. American Samoa, Guam, Northern Mariana Islands, and the U.S. Virgin Islands in addition to Puerto Rico). Would it be possible to add these territories as an additional feature implementation to shift_geometry down the line. Thank you so much again for this package! It would be another feather in the cap of R as part of a go-to ecosystem for our GIS analysts doing data science and making maps outside of ArcGIS/QGIS.

Sincerely, Benjamin Robinson.

**Comment by benjaminrobinson** [comment]
I spoke to the GIS Manager at my company and this is what she said!  She's very opinionated about our maps so take it with a grain of salt:

> That's awesome. We actually do it on our maps as a row of inset maps with labels.
> There were also two concerns with the map `shift_geometry` produced: 1, no labels for the inset and 2) no borders to indicate that there were inset maps. People recognize Hawaii and Alaska by shape. A significant portion might see PR and know what island it is. But once you get to the territories, no one is going to recognize them unless they are labelled.
> And because of that, if you don't have boundaries around indicating they are inset maps, people are going to think they represent islands and that is their actual location.

**Comment by walkerke** [comment]
@benjaminrobinson all good points!  However - those issues are probably beyond the scope of tigris and `shift_geometry()`.  `shift_geometry()` returns data; it is then up to the analyst to make the map with it.  Both ggplot2 and tmap have labeling functionality (e.g. https://ggplot2.tidyverse.org/reference/ggsf.html), as well as options for drawing boxes around features (e.g. https://github.com/Nowosad/spain-map-layout).

**Comment by jzadra** [comment]
Is there a way to filter out the hawaiian territories?  I only want the 50 states, and I typically just filter the state geoid as <60.

**Comment by walkerke** [comment]
@jzadra could you elaborate on the use-case a bit?  You want to drop Hawaii entirely, or drop the uninhabited archipelago as discussed above?   The current GitHub version of tigris solves the latter.  

```r
st <- tigris::states() %>%
  dplyr::filter(GEOID < "60") %>%
  tigris::shift_geometry()

plot(st$geometry)
```

![image](https://user-images.githubusercontent.com/4872833/145205377-58cc0bf1-8f2b-41e3-a42a-ba1ea07da6c4.png)

**Comment by jmanthc** [comment]
I've used this code to map Alaska - and it works great for making the state look normal (not all wonky), and I've successfully plotted counties and city polygons on the state using the tigris package. But I also want to be able to plot specific points using lat/long coordinates onto Alaska. Any advice on how I can do that? If I keep Alaska where it's supposed to be based on lat/long to do that [aka, NOT using tigris::shift_geometry()], the outline of the state looks distorted.
